js/lib/IString.js

```diff
@@ -1444,10 +1444,4 @@ IString.prototype = {
     }
 };
 
-Object.defineProperty(IString, 'length', {
-    get: function() {
-        return this._length();
```

Comment:
The _length() method on line 536 is now unused. Please remove that as well to clean up.

Reply:
I haven't done that because that would be a break change if someone is using it (I know it is private, but still...).
What do you think if we add the `deprecated` notation?

Reply:
Okay, good plan

Reply:
Thank you. Updated.